.github/workflows/linux-install.yaml

```
@@ -0,0 +1,54 @@
+name: Linux-Install
+run-name: Linux-Install
+on:
+  pull_request:
+    paths-ignore:
+    - README.md
+    - 'docs/**'
+    types: [ opened, reopened, synchronize ]
+    
+jobs:
+   Kokkos-040200-OpenMPI:
+    env:
+      KOKKOS_SRC: ${{ github.workspace }}/_deps/kokkos
+      KOKKOS_BUILD: ${{ github.workspace }}/_deps/kokkos-build
+      KOKKOS_INSTALL: ${{ github.workspace }}/_deps/kokkos-install
+      COMM_SRC: ${{ github.workspace }}
+      COMM_BUILD: ${{ github.workspace }}/build
+      COMM_INSTALL: ${{ github.workspace }}/install
+      COMM_INSTALL_BUILD: ${{ github.workspace }}/install-build
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+      - name: Install MPI
+        run: |
+          sudo apt-get update && sudo apt-get install -y libopenmpi-dev openmpi-bin cmake
+          which mpirun
+          mpirun --version
+          which mpicxx
+          mpicxx --version
+      - name: Check out repository code
+        uses: actions/checkout@v4
+      - name: List files in the repository
+        run: |
+          ls ${{ github.workspace }}
+      - name: Build Kokkos
+        run: |
+          git clone https://github.com/kokkos/kokkos.git "$KOKKOS_SRC"
+          cd "$KOKKOS_SRC" && git checkout 4.2.00
```

```suggestion
mkdir "$KOKKOS_SRC" && curl -sfL https://github.com/kokkos/kokkos/archive/refs/tags/4.2.01.tar.gz | tar xvz - --strip-components=1 -C "$KOKKOS_SRC"
```

Else we can clone only the last commit of the given tag.

---

I switched this to a shallow clone of the tag with `git clone --branch <tag> --depth 1` (the `--branch` flag is a bit misleading but it can accept a tag)